lib/reporter.js

```diff
@@ -245,31 +244,29 @@ var CoverageReporter = function (rootConfig, helper, logger, emitter) {
         // If config.check is defined, check coverage levels for each browser
         if (config.hasOwnProperty('check') && !checkedCoverage[browser.id]) {
           checkedCoverage[browser.id] = true
-          var coverageFailed = checkCoverage(browser, remappedCoverageMap)
-          if (coverageFailed) {
-            if (results) {
-              results.exitCode = 1
-            }
+          const coverageFailed = checkCoverage(browser, remappedCoverageMap)
+          if (coverageFailed && results) {
+            results.exitCode = 1
           }
         }
 
-        var context = istanbulLibReport.createContext(options)
-        var report = reports.create(reporterConfig.type || 'html', options)
+        const context = istanbulLibReport.createContext(options)
+        const report = reports.create(reporterConfig.type || 'html', options)
```

Comment:
Sorry I can't figure out what this commit is achieving.  I  *think* it is:
```
fix(reporter): update calls to  match new API in istanbul-lib-report v?
```

Reply:
I split refactoring and upgrade on separate PRs